apps/src/studio/skins.js

```diff
@@ -1668,6 +1668,13 @@ exports.load = function (assetUrl, id) {
   skin.dropdownThumbnailHeight = 50;
   skin.preloadAssets = true;
 
+  // Offset & dimensions of a rectangle in collidable in which wall collisions occur.
+  // For isometric-style rendering, this would normally be the feet.
+  skin.wallCollisionRectOffsetX = 0;
+  skin.wallCollisionRectOffsetY = 0;
+  skin.wallCollisionRectWidth  = 0;
+  skin.wallCollisionRectHeight = 0;
```

Comment:
a width and height of zero seems like an odd choice. Is this just intended to be temporary until we get art in? If so, I'd love to get a TODO comment in here